logistics_project/apps/api/resources/v0_1.py

```diff
@@ -68,12 +68,30 @@ class SupplyPointResources(ModelResource):
     active = fields.BooleanField('active')
     type = fields.CharField('type')
     location_id = fields.IntegerField('location_id', null=True)
+    products = fields.ToManyField(
+        ProductResources,
+        attribute=lambda
+            bundle: Product.objects.filter(
+                id__in=ProductStock.objects.filter(supply_point=bundle.obj,
+                                                   is_active=True,
+                                                   product__is_active=True).values_list(*['product'], flat=True),
```

Comment:
@czue 

This is not possible, because when we use ToManyField we must set Resource object, when we use ListField then we can't use callable function in '''attribute''' parameter.